### DISCUSSION TITLE
Copilot's actual model?

### DISCUSSION THREAD

**Discussion by andriy-koz** [community/community#113376] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hi! I've read somewhere that copilot is upgrading to GPT-4, that's true?
![copilot](https://github.com/community/community/assets/86450120/ec62284b-0571-408f-ab32-331e61c151c8)

**Comment by Akash1134** [comment, marked as answer]
Hi @andriy-koz ,

We upgraded the Copilot Chat experience in [November 2023](https://github.blog/changelog/2023-11-30-github-copilot-november-30th-update/), bringing more accurate and useful code suggestions with OpenAI’s GPT–4 model.

GitHub Copilot plugin use different OpenAI models for different use cases in the Copilot service (e.g. chat vs. code completion). It’s very likely that other models will prove better optimized for new, specialized use cases as we expand Copilot to serve more parts of the software development lifecycle. We are actively experimenting with new models for new user interaction models, and will continue to evaluate new technologies as they become available.

Hope that helps!

**Reply by andriy-koz** [reply]
That was clear, thank you!
